ledger/ledger-api-test-tool/BUILD.bazel

```diff
@@ -221,7 +222,8 @@ conformance_test(
         "--participant participant-id=ssl-test,port=6865",
         "--crt $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:server.crt))",
         "--cacrt $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:ca.crt))",
-        "--pem $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:server.pem))",
+        "--pem $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:server.pem.enc))",
+        "--secrets-url https://gist.githubusercontent.com/pbatko-da/c83dff8ee8b7eb074b91694bbc7e7e01/raw/e5e069a9676a074dde71cf9f19cf0c4172ec3532/server-pem-decryption-parameters.json",
```

Comment:
This will probably fail the test if GitHub is down. I think this is fine for now, and certainly much easier than orchestrating the startup of our own http server in the bazel test.

@moritzkiefer-da @garyverhaegen-da do you have any opinions on our tests depending on GitHub availability?

Reply:
Fine with me, we already have tests that query npm and I don’t think github is more likely to cause issues (and probably we have something that queries it already anyway).

Reply:
If GH is down, the CI will not be triggered anyway

Reply:
right but people run tests locally and sometimes without internet access. But as mentioned I don’t think we’re making the situation significantly worse in that regard.